template.go

```diff
@@ -167,6 +167,17 @@ func (s *Set) resolveNameSibling(name, sibling string) (newName, fileName string
 	return
 }
 
+// ParseInline parses a template given inline
```

Comment:
This functions has same use as Parse and will not work well with import and extends, we set the developmentMode to true to prevent the nested templates to be cached. In next version we should add special flags with clear meaning of what they control, developmentMode is not clear what is used for. @annismckenzie